import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -89,10 +89,16 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
       cstoreDicomwebAddr = flags.dicomwebAddr;
       cstoreDicomwebStowPath = flags.dicomwebStowPath;
     }
-    IDicomWebClient defaultCstoreDicomWebClient =
+    IDicomWebClient defaultCstoreDicomWebClient = null;
+    if (flags.useHttp2ForStow) {
+      defaultCstoreDicomWebClient =
         new DicomWebClientJetty(
             credentials,
             StringUtil.joinPath(cstoreDicomwebAddr, cstoreDicomwebStowPath));
+    } else {
+      defaultCstoreDicomWebClient =
+        new DicomWebClient(requestFactory, flags.dicomwebAddress);
```

Comment:
Just using flags.dicomwebAddress isn't backwards compatible with the old way of specifying flags.dicomwebAddr + flags.dicomwebStowPath which we used to concatenated together and then make a request here.

To make this backwards compatible we'd need to create the address as 
StringUtil.joinPath(cstoreDicomwebAddr, cstoreDicomwebStowPath), and then not append "/studies" in the stow function.

Reply:
Thanks for catching this. I think I have fixed it, but please take a close look.

I've filled https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/issues/85 to add an integration test for this in the future.